### DISCUSSION TITLE
Not sure if this is a bug or my code's issue

### DISCUSSION THREAD

**Discussion by vortexprime24** [vercel/next.js#74030] [Help] [answered]
### Summary

![image](https://github.com/user-attachments/assets/6ff3ac60-4c8d-4e15-9d3c-1b1a1cacc2df)

Look the image. Next.js spams "POST / 200 in <ms>" lot much like 10 times a second in the console and slows down the development preview as well as I cannot navigate to any routes when I click. After that a message came with error...I don't know what's wrong here

### Additional information

```js
Not sure
```

**Comment by icyJoseph** [comment]
Got a link to a repository?

**Reply by vortexprime24** [reply, marked as answer]
Uhh, I found out that it was fixed by removing a useless dependency added in useEffect() which was causing re-execution of code that fetches some data from somewhere. Figured it out!

**Comment by beachstrider** [comment]
It looks like a network issue.

**Reply by vortexprime24** [reply]
No.
